src/solver/optimisation/opt_rename_problem.cpp

```diff
@@ -1,12 +1,6 @@
 #include "opt_rename_problem.h"
 #include <sstream>
-void RenameLinkVariable(PROBLEME_ANTARES_A_RESOUDRE* problem,
-                        int var,
-                        Antares::Data::Enum::ExportStructDict structDict,
-                        int ts,
-                        Antares::Data::Enum::ExportStructTimeStepDict timeStepType,
-                        const std::string& origin,
-                        const std::string& destination)
+void VariableNamer::SetLinkVariableName(int var, Antares::Data::Enum::ExportStructDict structDict)
```

Comment:
About : **SetLinkVariableName(...)** (and other methods of class **VariableNamer**)
Why do we need an **ExportStructDict** to be passes as arg ?
Could we not use it internally and thus avoid this parameter ?